doc/user-manual/language/universe-levels.lagda.rst

```diff
@@ -84,6 +84,54 @@ With this definition, we have, for example:
     Nat x Set : Set₁
     Set × Set : Set₁
 
+Intrinsic level properties
+--------------------------
+
+.. _intrinsic-level-properties:
+
+Levels and their associated operations have some properties
+which are internally and automatically solved by the compiler.
+This means that we can replace some expressions with others,
+without worrying about the expressions
+for their corresponding levels matching exactly.
+
+For example, for some `F : (l : Level) -> Set l`, `l1 l2 : Level`,
+`G : {l : Level} -> Set l -> Set l -> Set l`
+if we have
+
+.. code-block:: agda
+
+    x : F (l1 ⊔ l2)
+
+.. code-block:: agda
+
+    y : F (l2 ⊔ l1)
+
+
+we can use
+
+.. code-block:: agda
+
+    G x y
+
+without having to do any additional work.
+
+
+Here's a list of these properties
```

Comment:
Punctuation: terminate with `:`.
Avoid colloquial style: "Here's" --> "Here is".